.github/workflows/test.yml

```diff
@@ -0,0 +1,151 @@
+name: Test
+
+on:
+  # Run on pushes to select branches and on all pull requests.
+  push:
+    branches:
+      - master
+      - trunk
+      - 'release/**'
+      - 'hotfix/[0-9]+.[0-9]+*'
+  pull_request:
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+# Cancels all previous workflow runs for the same branch that have not yet completed.
+concurrency:
+  # The concurrency group contains the workflow name and the branch name.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  integration:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        include:
+          - php_version: '5.6'
+            wp_version: '5.8'
+            multisite: true
+
+          - php_version: '7.0'
+            wp_version: 'latest'
+            multisite: false
+
+          - php_version: '7.2'
+            wp_version: '5.8'
+            multisite: false
+
+          - php_version: '7.3'
+            wp_version: 'trunk'
+            multisite: true
+
+          - php_version: '7.4'
+            wp_version: '5.8'
+            multisite: false
+
+          # WP 5.6 is the earliest version which (sort of) supports PHP 8.0.
+          - php_version: '8.0'
+            wp_version: '5.9'
+            multisite: false
```

Comment:
I suppose we could just remove the comment. I left it in for consistency with other workflow and as a "warning" in case the supported WP versions range would be widened, in which case it is important to ensure that the PHP - WP combi is a compatible one.

Reply:
I like consistency. And now I indeed read it as a warning that the `wp_version` shouldn't be set lower than 5.6. Thanks for clarifying that.